mockup of the Services page

june-andronick opened the pull request:
New services page.

june-andronick commented:
I think I'm done with this final draft. To note:
- Cog and DW have a separate page on the Foundation website
- Breakaway doesn't (but we haven't ask Benno yet; just took blurb from announcement)

Next step would be to write to the 3 of them with screenshots and ask for feedback. Given I'm away next week, sooner is better ;)

june-andronick commented:
Also: I haven't added AOS yet

gernotheiser commented:
I get a 404 on the preview link

lsf37 commented:
Preview link [here](https://htmlpreview.github.io/?https://github.com/seL4/website_pr_hosting/blob/PR_30/localhost/index.html)

gernotheiser commented:
Services page looks good (but I'd like AOS added before we go live ;-)
Also, the bullet list should link to the relevant sections.

The individual companies' pages they will probably want to change, now that the formatting is defined, they don't look so great. I suggest sending them not just a screen shot but the actual HTML source so they can play with it themselves.

lsf37 reviewed (approved):
Happy with the current state, pending approval by the members.

Furao commented on content/Foundation/Services/dornerworks.pml:
```diff
@@ -0,0 +1,96 @@
+      <td>
+
+         DornerWorks helps product makers turn their ideas into
+         realitywith FPGA, hardware, and embedded software engineering
```
add a space between reality and with

lsf37 replied:
Thanks for spotting these, now fixed!

Furao commented on content/Foundation/Services/dornerworks.pml:
```diff
@@ -0,0 +1,96 @@
+  </li>
+  </ul>
+</li>
+<li>
+  Guest OS and hardware portingVirtualization and hypervisor distributions
+</li>
```
Should be:

```
<li>
  Guest OS and hardware porting
</li>
<li>
  Virtualization and hypervisor distributions
</li>

lsf37 replied:
Done!

Furao reviewed (changes requested):
Just a couple of text issues. I appreciate the endorsement process page. That will be helpful.

lsf37 commented:
(reverted accidental push)

june-andronick commented:
I've added AOS as an endorsed training.
I've deleted commented out sections.
I changed the order and names in the drop-down menus. 
Cog and Breakaway have accepted the text for their parts.
This is ready for review and merge.
